src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1053,10 +1053,9 @@ else if (value instanceof BitSet) {
                 }
                 else {
                     result = Arrays.toString((int[]) value).replace("[", "").replace("]", "");
-
```

Comment:
Moved to the inner `if` block.

Reply:
Just wanted to confirm, so without this change we would instead see:
> `Property {@code limitedTokens} - Specify set of tokens with limited occurrences as descendants.`
> `Default value is .`

Reply:
Yes. The default value for `limitedTokens` in the xdocs was `<td/>`.
Now it is `<code>{}</code`